### PR TITLE
Added script node 2 way communication example 

### PR DESCRIPTION
**Pull request by Erol444**
using RawBuffer and ascii strings

### CODE REVIEW

File: script_node_communication/main/app_main.cpp [hunk 1]
```diff
@@ -15,54 +14,38 @@ extern "C"
...
+    std::cout << "String recieved from VPU: '" << retStr << "'" << std::endl;
 
-            }
-            // free up resources once you're done with the message.
-            mySpiApi.free_message(&nnMsg);
-            mySpiApi.spi_pop_messages();
-        } else {
-            // Periodically check if new messages are available
-            usleep(100);
-        }
```
Comment (themarpe) [inline script_node_communication/main/app_main.cpp:55]: Don't drop this - retain so polling is limited

**Review by themarpe — CHANGES_REQUESTED** [review]

File: script_node_communication/main/app_main.cpp (hunk 1 above)
Comment (Erol444) [reply script_node_communication/main/app_main.cpp:55]: what do you mean? On line 34 there's `sleep(1);`

Comment (themarpe) [reply script_node_communication/main/app_main.cpp:55]: If requesting message fails, then it'll loop right back and poll device again immediately. To reduce this polling, consider sleeping for a short amount of time in between

Comment (themarpe) [reply script_node_communication/main/app_main.cpp:55]: Nevermind missed the not operator there. LGTM

**Review by themarpe — APPROVED** [review]

**Pull request merged**